docs/rules/extensions.md

```diff
@@ -1,4 +1,6 @@
-# import/extensions - Ensure consistent use of file extension within the import path
```

Comment:
i'd prefer to see this tagline preserved somewhere

Reply:
A few of these rule docs had the rule description in their title. Note that these descriptions are all preserved in `meta.docs.description` which is used in the README rules table. If you wanted, we could include the rule descriptions in all the titles using the option [`--rule-doc-title-format`](https://github.com/bmish/eslint-doc-generator#--rule-doc-title-format) option. But for the most part, these descriptions are already captured in the rule doc bodies.

Reply:
I think it's useful for the exact phrase to appear somewhere in the doc - it doesn't have to be in the title ofc, but somewhere. Can that be automated/enforced somehow?

Reply:
Today, there are many plugins that include the rule description directly in the rule doc title, which is why we have [`--rule-doc-title-format desc-parens-prefix-name`](https://github.com/bmish/eslint-doc-generator#--rule-doc-title-format). That's probably the best/easiest way to achieve this within existing conventions.

That said, if that's still not satisfactory, then I'm thinking of some possible options that could achieve what you want.

| Potential Option | Description | Upside | Downsides |
| --- | --- | --- | --- | 
| `--rule-doc-header-include-description` | Whether to include the rule description below any notices at the bottom of the rule doc header (default `false`). | The rule description will be auto-generated in the rule doc header. | This isn't always the best place to put the rule description. Often, rule docs provide general context first, and then provide the description later (potentially in a "Rule details" section). |
| `--rule-doc-section-include-description` | Specifies the name of the section of the rule doc that should begin with the rule description. Pass option with no section name to include the description at the bottom of the rule doc header. | Ensures the rule description is mentioned in the appropriate section, often a "Rule details" section. This is a more flexible version of `--rule-doc-header-include-description`. | Most complex implementation.
| `--rule-doc-mention-description` | Whether to require the rule description to be mentioned somewhere in the rule doc (default `false`). | Allows greatest flexibility in placement. | Not auto-generated. The descriptions could end up placed in inconsistent positions. |

My recommendation in order of preference: 

1. Leave as-is. Rule docs already capture the spirit of the description even if the wording doesn't exactly match.
2. Include rule descriptions in the rule doc titles.
3. Add `--rule-doc-section-include-description` option.
4. Add `--rule-doc-mention-description` option.

Let me know what you think. If we want to implement a new option, then it would probably be best to [file an issue](https://github.com/bmish/eslint-doc-generator/issues/new) for this and avoid blocking the overall PR, since this only affected a few rule docs anyway.

Reply:
Hmm. I agree it needn't block the PR, regardless.

I think a simple tagline is important top-level context, so I'd prefer `--rule-doc-section-include-description`. Is that already implemented? If so, let's use it! If not, we can address it in the future.